modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -1574,6 +1574,22 @@ export const applications: ApplicationsNS = {
                                 empty: "Please fill the audience",
                                 invalid: "Please avoid special characters like commas (,)"
                             }
+                        },
+                        accessTokenAttributes: {
+                            hint : "Select the attributes that should be included in the <1>access_token</1>.",
+                            label: "Access Token Attributes",
+                            placeholder: "Search by attribute name",
+                            enable: {
+                                hint : "Previously, attributes that are set as requested in the User Attributes " +
+                                 "section were automatically added to <1>access_token</1>. Now, you can choose which " +
+                                 "user attributes are included in the <1>access_token</1>. To enable this new feature, " +
```

Comment:
Draft: "Enable this feature to return the selected attributes in the access token. By enabling this feature, any attribute that is marked as required will automatically be included in the access token even if the access token does not possess the necessary scopes."